rct229/rulesets/ashrae9012019/section5/section5rule39.py

```diff
@@ -0,0 +1,49 @@
+from rct229.rule_engine.partial_rule_definition import PartialRuleDefinition
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+
+
+class Section5Rule39(RuleDefinitionListIndexedBase):
+    """Rule 39 of ASHRAE 90.1-2019 Appendix G Section 5 (Envelope)"""
+
+    def __init__(self):
+        super(Section5Rule39, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, False, True),
+            each_rule=Section5Rule39.BuildingRule(),
+            index_rmr="proposed",
+            id="5-39",
+            description="Manually controlled dynamic glazing shall use the average of the minimum and maximum SHGC and VT.",
+            ruleset_section_title="Envelope",
+            standard_section="Section G3.1-5(a)5 Building Envelope Modeling Requirements for the Proposed design",
+            is_primary_rule=False,
+            list_path="ruleset_model_instances[0].buildings[*]",
+        )
+
+    class BuildingRule(RuleDefinitionListIndexedBase):
```

Comment:
Could you check if we need the building level? It seems like nothing is happening in this level.

Reply:
> Could you check if we need the building level? It seems like nothing is happening in this level.

Thanks, this rule is so simple I was second guessing how far I am able to skip ahead, or if maybe I have oversimplified.

Should the list_path for the Section5Rule39 class then skip straight to subsurfaces?